sdk/resource/config.go

```diff
@@ -92,3 +85,27 @@ type schemaURLOption string
 func (o schemaURLOption) apply(cfg *config) {
 	cfg.schemaURL = string(o)
 }
+
+// WithOS adds all the OS attributes to the configured Resource.
```

Comment:
If we are going to move these options to this file can we move all the `WithOS*` and `WithProcess*` options as well?